Tests/function/tests/Set-VSTeamVariableGroupVariable.Tests.ps1

```diff
@@ -0,0 +1,54 @@
+Set-StrictMode -Version Latest
+
+Describe 'VSTeamVariableGroup' {
+   BeforeAll {
+      . "$PSScriptRoot\_testInitialize.ps1" $PSCommandPath
+   }
+
+   Context 'Set-VSTeamVariableGroupVariable' {
+      Context 'Services' {
+         BeforeAll {
+            Mock _getApiVersion { return 'VSTS' }
+            Mock _getInstance { return 'https://dev.azure.com/test' }
+            Mock Get-VSTeamVariableGroup { @(@{
+               name = "TheGroup";
+               id = 101;
+               variables = @{
+                  "x" = @{value="A"};
+                  "y" = @{value="B"};
+                  "s" = @{value=$null; isSecret=$True}
+               }
+            }) | ConvertTo-JSON | ConvertFrom-JSON}
```

Comment:
Why are you doing a double convertion here?

Reply:
So that the output of the mocked Get...Group is PSObjects instead of hashtables, like the real Get...Group provides.